src/globalization.ts

```diff
@@ -724,10 +724,10 @@ export function parseNumber(value: string, style: string = "Number" || "Integer"
 	if (style === "Currency") {
 		result = _parseNumber(value.replace(cultureInfo.numberFormat.CurrencySymbol, ""), cultureInfo) * sign;
 
-		// if there is a decimal place, check the precision isnt greater than allowed for currency. 
+		// if there is a decimal place, check the precision isnt greater than allowed for currency.
 		// Floating points in js can be skewed under certain circumstances, we are just checking the decimals instead of multiplying results.
 		var resultvalue = result.toString();
-		if (resultvalue.indexOf('.') > -1 && (resultvalue.length - (resultvalue.indexOf('.') + 1)) > cultureInfo.numberFormat.CurrencyDecimalDigits	) {
+		if (resultvalue.indexOf(".") > -1 && (resultvalue.length - (resultvalue.indexOf(".") + 1)) > cultureInfo.numberFormat.CurrencyDecimalDigits	) {
```

Comment:
Apparently the current rule is to use double-quotes, which I realize is not consistent with the rules we use in other projects. I don't really care one way or another, but I'm hesitant to change it since it would show up as a violation all over the place, and to fix it we'd have to make lots of changes.